doc/faq.rst

```diff
@@ -133,6 +133,26 @@ configuration options, e.g. to specify a custom location for storing the
 datasets.
 
 
+A function uses multiple CPU cores even though I didn't tell it to. Why?
+------------------------------------------------------------------------
+Ordinarily in MNE-python the `parallel` module is used to deploy multiple
+cores via the `n_jobs` variable. However, functions like
+:func:`mne.preprocessing.maxwell_filter` that use :func:`numpy.linalg` do not have an
+`n_jobs` flag but may still use multiple cores. This is because `numpy.linalg`
+does not use `parallel`; instead it uses `OpenMP <http://www.openmp.org/>`_ via the `Intel Math Kernel
+Library (MKL) <https://software.intel.com/en-us/intel-mkl>`_. To control how many cores are used for functions like
```

Comment:
Do I need to reference OPENBLAS_NUM_THREADS as well, or just the library?  The former seems like overkill for the FAQ, since AFAIK the Q has never been A before.